### DISCUSSION TITLE
How to add border bottom for last row of table on each page in mpdf and extend last row to the footer

### DISCUSSION THREAD

**Discussion by parthibarath** [mpdf/mpdf#1874] [General]
I created the invoice with mpdf in laravel with HTML table with header and footer, if the table row reach the footer it auto breaks but it look like this
![FireShot Capture 086 -  - localhost](https://github.com/mpdf/mpdf/assets/67301613/791a6cd0-f297-4b31-91c6-7660e3eb1734)
but i need the border bottom of last row in each page how to do it?

And also i need to extend the last row height to start of the footer on each page.

**Comment by maker247** [comment]
I apologize for my previous comment
@jakejackson1 i feel ashamed of myself
